apps/explorer/src/ui/SplitPanes.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import clsx from 'clsx';
+import { Fragment, type ReactNode, useState } from 'react';
+import { Panel, PanelGroup, PanelResizeHandle } from 'react-resizable-panels';
+
+export type SplitPaneProps = {
+    panels: ReactNode[];
+    direction: 'horizontal' | 'vertical';
+    autoSaveId?: string;
+    defaultSizes?: number[];
+};
+
+function ResizeHandle({ isHorizontal }: { isHorizontal: boolean }) {
+    const [isDragging, setIsDragging] = useState(false);
+
+    return (
+        <PanelResizeHandle onDragging={setIsDragging}>
+            <div
```

Comment:
Should this also have a hover interaction so that this is more discoverable?

Reply:
added hover default state